lib/run-datadog.sh

```diff
@@ -21,22 +21,44 @@ start_datadog() {
     # so they must be grabbed separately
     datadog_tags=$(python $DATADOG_DIR/scripts/get_tags.py)
     sed -i "s/# tags:.*/tags: $datadog_tags/" $DATADOG_DIR/dist/datadog.yaml
-    legacy_datadog_tags=$(LEGACY_TAGS_FORMAT=true python $DATADOG_DIR/scripts/get_tags.py)
-    sed -i "s/# tags:.*/tags: $legacy_datadog_tags/" $DATADOG_DIR/dist/datadog.conf
-    sed -i "s~# log_file: TRACE_LOG_FILE~log_file: $DATADOG_DIR/trace.log~" $DATADOG_DIR/dist/datadog.conf
+    sed -i "s~log_file: TRACE_LOG_FILE~log_file: $DATADOG_DIR/trace.log~" $DATADOG_DIR/dist/datadog.yaml
```

Comment:
Not too fan of this way of passing config key to the agent, it seems very error-prone. It can probably wait but I'd like avoiding editing the configuration directly in the init script.

Reply:
@arbll if you have a better way of doing this I'm happy to hear it :)